### DISCUSSION TITLE
Обновление v3.23.3 убило сайт

### DISCUSSION THREAD

**Discussion by Xprinter-dv** [cdek-it/wordpress#47] [Q&A]
Не спешите обновляться, после установки обновления сайт завешивается при выборе варианта доставки до ПВЗ
Попытка откатить на старую версию вызвало задвоение плагина. Теперь в списке их два с одинаковыми названиями.
Update: Долго боролся и давал шанс плагину, но видимо нужно ещё годик подождать, пока работа станет более стабильна. Пришлось вернутся к платному плагину от стороннего разработчика.

**Comment by DEMAxx** [comment]
Добрый день! Не могли бы вы дать больше информации о проблеме?  
Из сообщения

>  сайт завешивается при выборе варианта доставки до ПВЗ

 не понятно на какой странице это происходит.
 
Если вы отправите скрин со страницы где происходит проблема или ошибки - это поможет нам решить проблему как можно быстрее.

**Comment by Xprinter-dv** [comment]
Была версия v3.22.3 работало нормально.
После обновления на v3.23.3 пробую оформить заказ, добавляю в корзину, перехожу на страницу оформления заказа, выбираю город, поумолчанию стоит доставка СДЭК курьером, переключаю на СДЭК до ПВЗ все доставки "мутнеют" в ожидании обновления и остаются в таком виде бесконечно, другую доставку уже выбрать невозможно. Если обновить страницу она не прогружается полностью.
Прислать скрин сейчас уже нет возможности проблема решилась удалением плагина и установкой другого от стороннего производителя.

Эта проблема оказалась уже критичной, потеря клиентов которые не могли заказ оформить и купили в другом месте.
О нескольких других проблемах сообщал на других темах.

**Comment by vermorag** [comment]
Добрый день, версия плагина 3.23.3 была отозвана из маркета и сейчас последняя актуальная - [3.22.3](https://github.com/cdek-it/wordpress/releases/tag/v3.22.3).

В обновлении 3.23.х были изменения, который должны были упростить функционал для пользователя при создании заказа: добавили все доступные печатные формы для ШК и переделали автоматизацию создания заказа, чтобы быть совместимыми с любыми платежными системами и автоматически создавать заказ даже если что-то пошло не так и вам пришлось вручную поменять статус заказа на "оплачен". Поэтому торопились, хотели уменьшить срок выпуска релиза, чтобы вы скорее могли им воспользоваться, однако не учли все ситуации и потеряли работоспособность функционала. Поправим нашу тестовую среду, добавим больше тестирования во избежание критических ошибок в дальнейшем. Приносим извинения за причиненные неудобства и заранее благодарим за понимание.
